recipes/hpo-toolkit/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "hpo-toolkit" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b137d190539767f1daf5e9f51db78e79bb0a57ffa564d55714e4af2633764535
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - networkx
+
+test:
+  imports:
+    - hpotk
+  requires:
+    - tox
+    - setuptools
+    - pip
+  commands:
+    - pip check
+about:
+  home: https://github.com/TheJacksonLaboratory/hpo-toolkit
+  summary: 'Simple & Efficient Hpo Toolkit'
+  license: GPL-3.0
+  license_file: LICENSE
+  description: |
+    hpo-toolkit is a simple and efficient library for mapping loinc test results
+    to hpo terms. Pure python, 3.8+
+
+extra:
+  recipe-maintainers:
+    - ielis
```

Comment:
@ielis Are you happy to be a maintainer?

Reply:
Hi @BastianZim , yes, I am super happy to be a maintainer of the package! Thanks a lot!